examples/sample_runtime/src/main.rs

```diff
@@ -313,7 +311,7 @@ fn main() {
             spec: Vec::default(),
         };
         let tx = service_keypair.request_artifact_deploy(SUPERVISOR_INSTANCE_ID, request);
-        api_sender.broadcast_transaction(tx).unwrap();
+        api_sender.broadcast_transaction(tx).wait().unwrap();
```

Comment:
`ApiSender` now provides async broadcast. This is because it's easy to go from async to sync (e.g., by using `wait()` as here), but not the other way around.